manifests/params.pp

```diff
@@ -11,9 +11,12 @@
   $disable_ssl               = false
   $open_ssl_listen_port      = undef
   $postgres_listen_addresses = 'localhost'
-
-  $database                  = 'postgres'
+  
+  $puppetdb_version          = $puppetdb::globals::version
+  $database                  = $puppetdb::globals::database
   $manage_dbserver           = true
+  $manage_pg_repo            = false
```

Comment:
Shall we do this as part of this PR? Or you wanna create a separate one? Also there is that bug with the manage_repos logic not taking a version if it isn't specified, I can't recall if that was done for a reason.